test/browser/element.vdiff.js

```diff
@@ -1,6 +1,6 @@
 import { css, html, LitElement, nothing } from 'lit';
 import { defineCE, expect, fixture, hoverElem } from '../../src/browser/index.js';
-import { executeServerCommand } from '@web/test-runner-commands';
+//import { executeServerCommand } from '@web/test-runner-commands';
```

Comment:
TODO

Reply:
Yeah I'll add this back after merging the goldens PR 👍